backend/__init__.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import os
+
+
+db_role = 'postgres'
+db_password = 'edison'
+db_url = '0.0.0.0'
+db_name = 'edison'
+
+# Put app and db here so the entire app could import them.
+app = Flask(__name__)
+basedir = os.path.abspath(os.path.dirname(__file__))
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://{}:{}@{}/{}'.format(db_role, db_password, db_url, db_name)
```

Comment:
This causes something that is a configuration value to be hard-coded in your application code. that can work in the short term, but will have to be modfied as you move to STAGE/PROD. Flask has several ways in which you can define a unified configuration file with separate sections for different environments. Please use one of them or create a ticket to do so in the near future.

Also consider that you're moving here from a standard way to define DB connections (The connection string) to a non-standard one (db_role/db_password/db_url/db_name), consider sticking to the standard way, it would make your application more portable.

Reply:
> Flask has several ways in which you can define a unified configuration file with separate sections for different environments.

WIP.

> Also consider that you're moving here from a standard way to define DB connections (The connection string) to a non-standard one (db_role/db_password/db_url/db_name), consider sticking to the standard way, it would make your application more portable.

Done.